src/hooks/swap/useSendSwapTransaction.tsx

```diff
@@ -42,7 +34,7 @@ export default function useSendSwapTransaction(
   account: string | null | undefined,
   chainId: number | undefined,
   provider: JsonRpcProvider | undefined,
-  trade: AnyTrade | undefined, // trade to execute, required
+  trade: InterfaceTrade | undefined, // trade to execute, required
```

Comment:
if it's required why allow undefined?

Reply:
I think this is in contrast to `trade?: InterfaceTrade`, which would allow undefined or unspecified.
I can remove this comment - I think it's just confusing.

Reply:
yea kinda unnecessary